src/webgl/p5.Camera.js

```diff
@@ -345,9 +345,22 @@ p5.prototype.frustum = function(...args) {
  * }
  *
  * function draw() {
+ *   background(0);
+ *   // The camera will automatically
+ *   // rotate to look at [0, 0, 0].
  *   camera.lookAt(0, 0, 0);
+ * 
+ *   // The camera will move on the
+ *   // x axis.
  *   camera.setPosition(sin(frameCount / 60) * 200, 0, 100);
  *   box(20);
+ *   
```

Comment:
oh sorry, one more thing, I think tests are failing because there are trailing spaces on this line

Reply:
and also line 352, I think? If you run `npm run lint` I believe it'll let you know when you've got all of them!

Reply:
Hmm... lines `352` and `357`. I removed the trailing spaces, let's see if this gets through!